### PR TITLE
PullRequest for Issue1355: Change the value of crystal Theta PV and the crystal Chi PV when crystal is changed

### PR DESCRIPTION
**Pull request by XuetaoLiu-CLS**
https://github.com/acquaman/acquaman/issues/1355

When crystal is changed, we move the crystal Theta and the crystal Chi to a given position for the convenience of the beamline staff

@dretrex @davidChevrier

### CODE REVIEW

**Comment by XuetaoLiu-CLS** [comment]
Test done

File: source/beamline/AMPVControl.cpp [hunk 1]
```diff
@@ -213,7 +213,7 @@ AMControl::FailureExplanation AMPVControl::move(double setpoint) {
 void AMPVControl::onNewFeedbackValue(double) {
 
 	// If we're not in the middle of a move, don't really care about changing values.
-	if(!moveInProgress_)
+	if(!moveInProgress_ && !completionTimer_.isActive())
```
Comment (dretrex) [inline source/beamline/AMPVControl.cpp]: Why is this necessary?

Comment (XuetaoLiu-CLS) [inline source/beamline/AMPVControl.cpp]: @dretrex for the crystal selection, it needs a long to move. The value of feedback will change twice during the process. When the final value is changed, the `moveInProgress_` is False already. But we still have the `completionTimer_` active (60s timout for this case). 

With this check, we will be able to have the complete signal ASAP. This is our original design as well (in the codes below, we stopped the timer. But it will only works when the move is quick enough).

Comment (dretrex) [inline source/beamline/AMPVControl.cpp]: There is another issue if your crystal selection move is taking too long.  We need to think about how these changes can affect other implementations and a general change to AMPVControl for a single exception does not seem like a good idea to me.
